wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md

```diff
@@ -2,32 +2,6 @@
 outdated: true
 ---
 
-# Traitement des erreurs
```

Comment:
why remove im not following

Reply:
changed according to this 

http://web.archive.org/web/20160819223414/http://osu.ppy.sh/wiki/Handling_Cheating/Foul_Play

looks like somehow two seperate articles were put inside a single file so i deleted the one that surely shouldn't be in *this* folder

Reply:
if its not copied from somewhere else on the wiki maybe throw it in meta/unused then

Reply:
done